config/src/main/java/org/springframework/security/config/annotation/web/configurers/WebAuthnConfigurer.java

```diff
@@ -127,26 +119,15 @@ public void configure(H http) throws Exception {
 		http.addFilterBefore(webAuthnAuthnFilter, BasicAuthenticationFilter.class);
 		http.addFilterAfter(new WebAuthnRegistrationFilter(userCredentials, rpOperations), AuthorizationFilter.class);
 		http.addFilterBefore(new PublicKeyCredentialCreationOptionsFilter(rpOperations), AuthorizationFilter.class);
+		http.addFilter(DefaultResourcesFilter.webauthn());
```

Comment:
We don't want to register the webauthn JavaScript unless the default log in page is being used otherwise a user cannot override the JavaScript. Users that still want the JavaScript an explicitly register the JavaScript.

We should probably update DefaultWebAuthnRegistrationPageGeneratingFilter to only be registered when the default log in page is enabled.